helios-services/src/main/java/com/spotify/helios/agent/AgentParser.java

```diff
@@ -272,6 +281,13 @@ protected void addArgs(final ArgumentParser parser) {
         .action(storeTrue())
         .setDefault(false)
         .help("If specified, the agent won't write job histories to ZooKeeper.");
+
+    secretVolumeManagerArg = parser.addArgument("--secret-volume-manager")
+        .help("Enable automatic procurement of secrets using this secret volume manager.");
```

Comment:
it's not clear from this help text that this is meant to be a `host:port` string